high_card.rb

```diff
@@ -1,5 +1,64 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+##
 
+
+def deck_of_cards
+  values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K', 'A']
+  suits = ['hearts', 'diamonds', 'clubs', 'spades']
+  deck = []
+
+  values.each_with_index do |v, i|
+    suits.each do |s|
+      deck.push({
+        score: i,
+        value: v,
+        suit: s,
+      })
+    end
+  end
+
+  return deck.shuffle
```

Comment:
Recommend separating deck.shuffle from creation of deck.  Create once.  Shuffle many times.